spec/models/user_spec.rb

```diff
@@ -0,0 +1,47 @@
+RSpec.describe User, type: :model do
+  it 'should have valid Factory' do
+    expect(FactoryBot.create(:user)).to be_valid
+  end
+
+  describe 'Validations' do
```

Comment:
Couple of issues for these tests:
- prefer using `expect` to `should`
- Checkout [shoulda-mathers gem](https://github.com/thoughtbot/shoulda-matchers). Very helpful for model tests
- So much repetition of user params. Leverage FactoryBot's `attributes_for`